unit_testing/fakes/fake_stm32xxxx_hal.h

```diff
@@ -16,10 +16,8 @@
 ** reducing the amount of duplication this file requires */
 #include "fake_stm32f401xc.h"
 #define STM32F401xC
```

Comment:
We should add the entry for the H7 as well

Reply:
I agree, but since it works, lets add that when we get there